client/src/modules/employees/registry/search.modal.js

```diff
@@ -71,13 +71,19 @@ function EmployeeRegistryModalController(ModalInstance, Functions, Grades, bhCon
   // returns the parameters to the parent controller
   function submit(form) {
     if (form.$invalid) { return; }
-    
+
     // push all searchQuery values into the changes array to be applied
     angular.forEach(vm.searchQueries, function (value, key) {
       if (angular.isDefined(value)) {
         changes.post({ key : key, value : value });
       }
     });
+    // let push defaultQueries values into the filter
```

Comment:
I wonder why no other modules do this, yet their filters are still applied.  That's curious.